P5/Source/Specs/sequence.xml

```diff
@@ -19,7 +19,8 @@
     </constraint>
   </constraintSpec>
   <attList>
-    <attDef ident="preserveOrder">
+    <attDef ident="preserveOrder" validUntil="2025-03-15">
```

Comment:
Given that a) `<interleave>` did not make it into the recent release, and b) use of `@preserveOrder` still works in the Stylesheets, I think the `@validUntil` date should be moved up to late 2025 or early 2026.